bolt_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 	"golang.org/x/sys/windows"
 )
 
+func init() {
+	vectorizedWritesAvailable = false
```

Comment:
also not sure whether this is the best way to handle the availability of this feature